ui/component/core/src/index.ts

```diff
@@ -109,6 +109,34 @@ export enum MapType {
     RASTER = "RASTER"
 }
 
+//todo: fix the ts problem
+export interface AttributeMarkerColours {
+    type: "string" | "boolean";
+    [value: string]: string;
+    //@ts-ignore
```

Comment:
@DonWillems wished for labels to be toggled, but obviously we can't do it this way because of line 115 in this file.
Was in a rush now so couldn't really find a solution to prep the model for this feature.